install.sh

```diff
@@ -401,25 +403,33 @@ function create_directories {
 	$sudo_cmd chown -R $MA_USER:$MA_GROUP $APP_INSTALL_DIR
 }
 
-function create_users_and_groups {
+function create_users {
 	log_info "Creating system group(user): $MA_GROUP($MA_USER)"
 	getent passwd "$MA_USER" >/dev/null ||
 		$sudo_cmd adduser --system --group --home $APP_INSTALL_DIR --shell /sbin/nologin "$MA_USER" &&
 		{ $sudo_cmd usermod -L "$MA_USER" ||
 			log_warn "Cannot lock the 'metrika-agent' user account"; }
+}
+
+function add_user_groups {
+	if [ "$NO_SYSTEMD_JOURNAL_GRP_REQUESTED" -ne 1 ]; then
+		# enable journald access for log tailing if not explicitly disabled
+		log_info "Adding $MA_USER to systemd-journal group to allow journal log tailing."
 
-   	# enable journald access for log tailing
-   	$sudo_cmd usermod -aG systemd-journal $MA_USER
+		$sudo_cmd usermod -aG systemd-journal $MA_USER
```

Comment:
I remember that I've experienced this thing with adding my user to docker for personal install, it would not always work and I'd need to use `newgrp` command - check step 3 [here](https://docs.docker.com/engine/install/linux-postinstall/). Perhaps we want to add it too.

Reply:
You'd normally need to that if you apply group membership to your user manually and you'd like to use the membership in the same login shell https://man7.org/linux/man-pages/man1/newgrp.1.html.

When installing the agent, most commonly the user running the installation bash script will not start the agent in the same login shell so I'd argue we don't need to call `newgrp(1)` here.